### PR TITLE
Absolute Pain (Doubles the SMES input / output

### PR DESCRIPTION
**Pull request by StrangeWeirdKitten**
I can not state how heavily fucked our powergrid is right now. So here's a bandaid solution.

## Changelog

:cl:
balance: doubles SMES input / output
/:cl:

### CODE REVIEW

**Comment by destrucktoid** [comment]
Have you also made it so that APC arcing's power requirement is doubled?

**Comment by StrangeWeirdKitten** [comment]
> Have you also made it so that APC arcing's power requirement is doubled?

It was already done by LT3

**Comment by lessthnthree** [comment]
Increasing SMES input/output was something I was intending to propose upstream at TG but didn't manage before vacation.

File: modular_zubbers/code/game/machinery/smes.dm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/obj/machinery/power/smes
```
Comment (lessthnthree) [inline modular_zubbers/code/game/machinery/smes.dm:1]: However, don't do it like that.

Instead, edit code\modules\power\smes.dm Line 67
```suggestion
/obj/machinery/power/smes/RefreshParts()
	SHOULD_CALL_PARENT(FALSE)
	var/power_coefficient = 1 // BUBBER EDIT CHANGE 
```

Comment (StrangeWeirdKitten) [reply modular_zubbers/code/game/machinery/smes.dm:1]: 👍

**Comment by Redrover1760** [comment]
Could you double SMES starting power as well? It runs out so fast and its been agony for every single shift ever without an immediate competent engineering spawning roundstart and rushing the engine.

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**